src/backend/_scss/application.scss

```diff
@@ -129,7 +129,7 @@ a {
 
 /* Navbar adjustments */
 .logo {
-  max-height: 45px;
```

Comment:
@Princu7 If the width of image is < max-width or height of image < max-height.
Max-width and Max-height will not be considered to calculate `aspect-ratio` as it is between width/height.